client/src/constants.ts

```diff
@@ -1,9 +1,21 @@
+import { ChainDefaults } from "./types";
+
 export const ClientConstants = Object.freeze({
   MIN_MAX_FEE_PER_GAS: 5000000000n,
```

Comment:
We will need to make this chain-dependent as well, since this is too high for OP Stack chains.